internal/utils/video.go

```diff
@@ -72,22 +67,23 @@ func GetVideoStartTimeDurFrames(segment []byte) (time.Duration, time.Duration, i
 	var keyFrames int
 	var lastKeyFramePTS time.Duration = -1
 	if strms, err := demuxer.Streams(); err == nil {
-		// glog.V(model.VERBOSE).Infof("=======--- streams: %+v", strms)
-		// glog.Infof("=======--- streams: %+v", strms)
 		for i, s := range strms {
 			if s == nil {
 				continue
 			}
 			if s.Type().IsVideo() {
-				videoIdx = int8(i)
-				break
+				if videoIdx == 0 {
+					videoIdx = int8(i)
```

Comment:
Perhaps init `videoIdx` with `-1` so we can differentiate no video from video on first position? Or is that already handled in some way?

Reply:
Yeah, good point, updated.